codegen/type_converter.go

```diff
@@ -604,6 +650,10 @@ func (c *TypeConverter) genStructConverter(
 			fromIdentifier = "in." + fromPrefix + pascalCase(fromField.Name)
 		}
 
+		if prevKeyPrefixes == nil {
+			prevKeyPrefixes = []string{}
```

Comment:
saw you sometimes use `[]string{}`, sometimes `make`, what's the difference? would be nice to have some consistency if no essential difference

Reply:
I may have missed a line, but I think I am using []string{} for arrays and make() for maps